ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -343,6 +343,11 @@ short getVersion() {
                 indexFile.getAbsolutePath(), startOffset, keyToFind);
           }
           // binary search on the mapped file
+          if (mmap.isLoaded()) {
+            metrics.indexSegmentInMemory.inc();
```

Comment:
probably more meaningful to name these as `indexSegmentInMemoryForFind` and `indexSegmentNotInMemoryForFind`